web/src/components/login/LoginForm.tsx

```diff
@@ -21,8 +22,10 @@ const LoginForm: FC = () => {
 
   const onSubmit = (data: any) => {
     const { email, password } = data;
-
+    tokenStore.loginData.email = email;
+    tokenStore.loginData.password = password;
```

Comment:
We should not save password in state! Anyone can view state values via dev tools...